packages/plugin-koa/src/koa.js

```diff
@@ -0,0 +1,64 @@
+const createReportFromErr = require('@bugsnag/core/lib/report-from-error')
+const extractRequestInfo = require('./request-info')
+
+const handledState = {
+  severity: 'error',
+  unhandled: true,
+  severityReason: {
+    type: 'unhandledErrorMiddleware',
+    attributes: { framework: 'Koa' }
+  }
+}
+
+module.exports = {
+  name: 'koa',
```

Comment:
The convention I've used for `plugin.name` is `camelCase` – though there aren't many (any?) plugins so far that have more than one word in their name, so you'd never know!

The other usage is in the `severityReason` part of the payload, which is a human-readable string rather than an identifier – which is why it's capitalized there.

Reply:
Ah ok I think I see now. The inconsistency I saw was in relation to this line https://github.com/bugsnag/bugsnag-js/compare/u/node-global-error-handlers...u/koa-plugin#diff-e82ce309a31234c4c5c4ee751c80a643R9